test/extended/machines/workers.go

```diff
@@ -122,64 +107,65 @@ var _ = g.Describe("[Feature:Machines][Disruptive] Managed cluster should", func
 		skipUnlessMachineAPIOperator(c.CoreV1().Namespaces())
 
 		g.By("validating node and machine invariants")
-		// fetch machines
-		machines, err := listMachines(dc, machineLabelSelectorWorker)
+		// fetch all machines
+		allMachines, err := listMachines(dc, "")
```

Comment:
No need to signify the variable contains all machines. The original `machines` will suite well as well.